CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Next release
 
+### Allow access to session variables by computed fields (fix #3846)
+
+Sometimes it is useful for computed fields to have access to the Hasura session variables directly.  Support for this is now added through the `add_computed_field` API.
```

Comment:
Can we give an example? One such example is available in the original issue.

Reply:
Okay, I'll add a quick example here.